hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneClientMultipartUploadWithFSO.java

```diff
@@ -311,6 +315,118 @@ public void testUploadPartOverrideWithRatis() throws IOException {
         commitUploadPartInfo.getPartName());
   }
 
+  @Test
+  public void testUploadTwiceWithEC() throws IOException {
+
+    String volumeName = UUID.randomUUID().toString();
+    String bucketName = UUID.randomUUID().toString();
+    String keyName = UUID.randomUUID().toString();
+    StringBuilder sb = new StringBuilder("sample Value");
+    for (int i = 0; i < 4096; ++i) {
+      sb.append("01234567890123456789");
+    }
+    String sampleData = sb.toString();
+
+    store.createVolume(volumeName);
+    OzoneVolume volume = store.getVolume(volumeName);
+    OzoneBucket bucket = getOzoneECBucket(volumeName, bucketName);
+
+    // perform upload and complete
+    OmMultipartInfo multipartInfo = bucket.initiateMultipartUpload(keyName,
+        bucket.getReplicationConfig());
+
+    String uploadID = multipartInfo.getUploadID();
+    int partNumber = 1;
+
+    OzoneOutputStream ozoneOutputStream = bucket.createMultipartKey(keyName,
+        sampleData.length(), partNumber, uploadID);
+    ozoneOutputStream.write(string2Bytes(sampleData), 0, sampleData.length());
+    ozoneOutputStream.close();
+
+    OmMultipartCommitUploadPartInfo commitUploadPartInfo = ozoneOutputStream
+        .getCommitUploadPartInfo();
+
+    String partName = commitUploadPartInfo.getPartName();
+    Map<Integer, String> partsMap = new HashMap<>();
+    partsMap.put(partNumber, partName);
+    bucket.completeMultipartUpload(keyName, uploadID, partsMap);
+
+    Assert.assertEquals(volume.getBucket(bucketName).getUsedBytes(), 137228);
```

Comment:
Where does "137228" come from? How it is derived?

Reply:
I guess its the EC data at 1024 chunk size stripped across the 3 nodes and 2 parity. But I think we need a comment or a calculation here to show were it comes from. Otherwise its just a magic number that does make sense to anyone in the future.

Reply:
@sodonnel Updated...